src/transformStateWithClones.js

```diff
@@ -7,7 +7,29 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const output = [];
+  let currState = { ...state };
+
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
```

Comment:
Move to the object. Example of usage: `case AtcionsTypes.AddProperties`

Reply:
could you be more specific please?

Reply:
Mentor ask you to write object 
```
const AtcionsTypes = {
  AddProperties: 'addProperties',
  .......
}
```

It will be like constant, you can reuse it and don't worry about typo